core/tests/test_utils.py

```diff
@@ -483,12 +483,32 @@ def get_expected_logout_url(self, slug):
         """Returns the expected logout URL."""
         return current_user_services.create_logout_url(slug)
 
-    def get_response(self, url, params=None, expect_errors=False,
-                     expected_status_int=200,
-                     expected_content_type='text/html'):
-        """Get a response, transformed to a Python object."""
+    def get_response(
```

Comment:
Make private (by prepending underscore)? That ensures it doesn't get used elsewhere -- only by get_html_response etc.

Reply:
`get_response` is being used by other functions where the content-type is different than html or json -- for example png or text/plain.

Reply:
Ah, I see. Then, I suggest: make get_response private, and define a new get_custom_response that first checks that the content type is not html/json, and that then calls _get_response(). External callers should use get_custom_response(). I think that would be a cleaner API.

Reply:
done